<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8"/>

	<link href="/css/style.css" type="text/css" rel="stylesheet">
	<script src="https://code.jquery.com/jquery-latest.min.js"></script>

</head>

<body>
	<div id="load-header">
	</div>
	<script type="text/javascript">
		$("#load-header").load("https://ltaylor2.github.io/index.html #header",
			function() {
				var links = document.getElementsByClassName("menu-item");
				for (var l = 0; l < links.length; l++) {
					links[l].classList.remove("menu-item-active");
				}
		});
	</script>

	<div class="page">
		<div id = "akatulo-content">
			<h3> 
				In this future, this page will host links to data, code, and publication information for the <em>Etheostoma akatulo</em> project. For now, it simply lists references and acknowledgments for
				the Evolution2019 conference poster.
			</h3>

			<h3>METHODS CLARIFICATION</h3>
			<p>
				All 2017 and 2018 <em>E. akatulo</em> samples were collected by J.W. Simmons (Tennessee Valley Authority). All fin clips were removed from the upper lobe of the caudal fin, and no individuals &le;30 mm were sampled due to mortality concerns. L.U.T. was supported by a National Science Foundation Graduate Research Fellowship (DGE1752134).
			</p>
			<h3>Acknowledgements</h3>
			<p>
			Our sincere thanks to the designers of Ipyrad (http://github.com/dereneaton/ipyrad), especially to Isaac Overcast, for streamlining the filtering and assembly of RAD-seq data. The beautiful fish drawings are sourced from www.americanfishes.com, with credit to Joseph Tomelleri for all species except for <em>E. stigmaeum</em>, which was drawn by Stephen T. Toss.
			</p>
			<h3>References</h3>
			<p>
				Frichot, E., and O. François. 2015. LEA: an R package for landscape and ecological association studies. Methods in Ecology and Evolution 6:925–929.
			</p>
			<p>
				Frichot, E., F. Mathieu, T. Trouillon, G. Bouchard, and O. François. 2014. Fast and efficient estimation of individual ancestry coefficients. Genetics 196:973–983.
			</p>
			<p>
				McCartney-Melstad, E., M. Gidiş, and H. B. Shaffer. 2019. An empirical pipeline for choosing the optimal clustering threshold in RAD seq studies. Molecular ecology resources.
			</p>
			<p>
				Nguyen, L.-T., H. A. Schmidt, A. von Haeseler, and B. Q. Minh. 2014. IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Molecular biology and evolution 32:268–274.
			</p>
			<p>
				Pickrell, J. K., and J. K. Pritchard. 2012. Inference of population splits and mixtures from genome-wide allele frequency data. PLoS genetics 8:e1002967.
			</p>
			<p>
				R Core Team. 2018. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. 2016.
			</p>
			<p>
				Robinson, J. D., J. W. Simmons, A. S. Williams, and G. R. Moyer. 2013. Population structure and genetic diversity in the endangered bluemask darter (<em>Etheostoma akatulo</em>). Conservation Genetics 14:79–92.
			</p>
			<p>
				Wickham, H. 2017. Tidyverse: Easily install and load 'tidyverse' packages. R package version 1.
			</p>
			<p>
				Yu, G., D. K. Smith, H. Zhu, Y. Guan, and T. T.-Y. Lam. 2017. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution 8:28–36.
			</p>
		</div>
	</div>
</body>
</html>